### ISSUE TITLE
Syntax highlighting CSS is broken?

### ISSUE THREAD

**Issue by puresyntax71** [puresyntax71/hugo-theme-chunky-poster#32]
This is from [gitlab](https://gitlab.com/virtualcursor/hugo-theme-chunky-poster):

> First, thank you so much for this fantastic theme. Our lab needs a quick but professional looking website that's more than a minimal blog, so thank you!
>
> In both the demo website as well as my own installation (NB: I have correctly copied all content/ from exampleSite, so unlike the live demo, images, etc. show up correctly), the CSS in the syntax highlighting shows up with a sort of offset drop-shadow. See it here:
>
> https://themes.gohugo.io//theme/hugo-theme-chunky-poster/post/markdown-syntax/
>
> under "Code block with Hugo's internal highlight shortcode". While working on this issue, I found that in the `exampleSite/` config.toml, the prismJS theme was left blank (`""`). By filling this in the problem was resolved. Thus I suggest that a theme be filled in in the example `config.toml`.
>
> *However*, I also found that disabling prismJS and enabling `codeFences` also did not work properly, which must be a CSS issue?
>
> Thanks again and kind regards

**Comment by puresyntax71** [comment]
Yeah, as [mentioned](https://gitlab.com/virtualcursor/hugo-theme-chunky-poster/issues/1#note_268791695), I've loaded the [default](https://github.com/puresyntax71/hugo-theme-chunky-poster/blob/master/layouts/partials/head.html#L17) when no theme is set but prismJS is enabled.

Might want to re-test it. I think I remember cdnjs reporting 404 for the theme CSS some few days ago although this seems to be working now. Can check it on the demo hosted on [netlify](https://hugo-theme-chunky-poster.netlify.com/post/markdown-syntax/). The one on Hugo themes doesn't use prismJS.

> *However*, I also found that disabling prismJS and enabling `codeFences` also did not work properly, which must be a CSS issue?

I'll check this out later on. Maybe can just generate a CSS styling and include as a custom CSS.

Thanks again!

**Comment by jblachly** [comment]
Same issue shows on the netlify hosted site as well. Image of the doubled / ghosted letters attached; note that this also exhibits the issue (and screenshot shows) that I don't believe I mentioned whereby the code blocks indented with four spaces also have a funky background.

https://imgur.com/a/6bH4Idy

**Comment by puresyntax71** [comment]
Yeah, I have only added support for `codeFences` ("Code block with backticks"). I'll try to add support for prismJS for indented code but probably not for Hugo's shortcode. Think I'll just leave the shortcode but will update the README on how to include the CSS for it.

Might work on this tonight.

Thanks!

**Comment by jblachly** [comment]
Great, also mention in the README that simply specifying a prismJS theme will fix the appearance even in the shortcode block.

Thanks again for the theme, it really is the best looking and functional combination of blog/non-blog content for an academic lab

**Comment by puresyntax71** [comment]
> Great, also mention in the README that simply specifying a prismJS theme will fix the appearance even in the shortcode block.

Yeah, no need anymore. I've added some changes using Javascript to try and fix this. The default theme doesn't seem to successfully override the styles added by Hugo itself while the other themes seem to do so.

> Thanks again for the theme, it really is the best looking and functional combination of blog/non-blog content for an academic lab

Glad you like it!

Currently there isn't a simple way to add custom CSS other than overriding the partial `head.html` itself. When setting `noClasses` to `false` and generating a [syntax CSS](https://gohugo.io/content-management/syntax-highlighting/#generate-syntax-highlighter-css), might need to include the CSS in the overridden partial.

Feel free to check the [demo](https://hugo-theme-chunky-poster.netlify.com/post/markdown-syntax/) again. I've also updated the README since I've added support for `codeFences` even when it is enabled.
